includes/init_includes/init_customer_auth.php

```diff
@@ -180,9 +181,9 @@
    * customer must be logged in to browse
    * customer is logged in and changed to must be authorized to browse
    */
-  if (!in_array(zcRequest::readGet('main_page'), array(FILENAME_LOGIN, FILENAME_LOGOFF, FILENAME_CONTACT_US, FILENAME_PRIVACY))) {
-    if (zcRequest::readGet('main_page') != CUSTOMERS_AUTHORIZATION_FILENAME) {
-      zen_redirect(zen_href_link(preg_replace('/[^a-z_]/', '', CUSTOMERS_AUTHORIZATION_FILENAME)));
+  if (!in_array($zcRequest->readGet('main_page'), array(FILENAME_LOGIN, FILENAME_LOGOFF, FILENAME_CONTACT_US, FILENAME_PRIVACY))) {
+  if ($zcRequest->readGet('main_page') != CUSTOMERS_AUTHORIZATION_FILENAME) {
+    zen_redirect(zen_href_link(CUSTOMERS_AUTHORIZATION_FILENAME));

```

Comment:
I'd like to see the regex filter restored to this, ie: `preg_replace('/[^a-z_]/', '', CUSTOMERS_AUTHORIZATION_FILENAME)` ... because that's a constant whose value is read from userland-input stored into db.